Tests/FunctionalTests/AEPEdgeFunctionalTests.swift

```diff
@@ -32,13 +32,17 @@ class AEPEdgeFunctionalTests: TestBase {
     #elseif os(tvOS)
     private let EXPECTED_BASE_PATH = "https://ns.adobe.com/experience/mobilesdk/tvos"
     #endif
+    
+    private var networkService: MockTestNetworkService = MockTestNetworkService()
```

Comment:
We can remove the initialization here since it is being done in the setup.

Reply:
few options:
1. Make networkService optional 
`private var networkService: MockTestNetworkService`
2. Add init block
```
 init {
 networkService = MockTestNetworkService()
}
```